### PR TITLE
Update the ABI dumps.

### PR DESCRIPTION
**Pull request by coryan**
We decided to update the ABI dumps to learn more about how these tools
work. I updated the ABI dumps for google_cloud_cpp_common,
storage_client (was missing) and bigtable_client. These files are huge,
so I am committing them compressed, that also keeps the line number
stats more sensible.

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/2166)

### CODE REVIEW

**Comment by coryan** [comment]
In case we are interested in the output, here is the (text formatted) API report for the latest changes:

https://travis-ci.com/googleapis/google-cloud-cpp/jobs/183088441#L4097


The specific section around source-level compatibility is here:

https://travis-ci.com/googleapis/google-cloud-cpp/jobs/183088441#L4637

I found some of the source level reports interesting:

https://travis-ci.com/googleapis/google-cloud-cpp/jobs/183088441#L5392

**Comment by coryan** [comment]
PTAL. We said we would be interested in the results of the ABI checker.  Here are some:

https://travis-ci.com/googleapis/google-cloud-cpp/jobs/183088441#L4097

for the changes since more or less yesterday.  I think the formatting as text is kind of terrible, it would be much better if we could capture the HTML output and host that somewhere (I believe we can do that with Kokoro).

In any case, the report tells us about stuff we changed (changed a bunch of return types, added some functions, etc.) Which I think would be nice to know about.

File: ci/check-abi.sh [hunk 1]
```diff
@@ -31,14 +31,15 @@ if [ "${CHECK_ABI:-}" != "yes" ]; then
 fi
 
 exit_status=0
-for library in google_cloud_cpp_common bigtable_client; do
+for library in storage_client google_cloud_cpp_common bigtable_client; do
```
Comment (devbww) [inline ci/check-abi.sh]: Q. Is there an ordering we might prefer (e.g., google_cloud_cpp_common first)?

Comment (coryan) [reply ci/check-abi.sh]: I cannot think of any order that is much better than any other, so I am going with alphabetical for now.  We can change it if we learn this is not ideal.

**Pull request merged**